cmd/bpfman-operator/main.go

```diff
@@ -46,9 +49,33 @@ var (
 func init() {
 	utilruntime.Must(clientgoscheme.AddToScheme(scheme))
 	utilruntime.Must(bpfmaniov1alpha1.Install(scheme))
+	utilruntime.Must(osv1.Install(scheme))
 	//+kubebuilder:scaffold:scheme
 }
 
+func getPlatformInfo(client discovery.DiscoveryInterface, cfg *rest.Config) (bool, error) {
```

Comment:
nit: This function should be renamed or add a comment. Returns true if Openshift, false otherwise.

Reply:
I'll do this!